validation/schema.json

```diff
@@ -40,6 +40,29 @@
     "details": {
       "type": "string"
     },
+    "severity": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "CVSS_V3",
+              "PLAIN_TEXT",
+              "OTHER"
```

Comment:
OTHER is not in the documentation. remove?

Reply:
Oh, huh. Not sure what I had in mind when adding that. Will remove it.